File: api/springboot/src/main/java/com/persona/apipersona/domain/Comentario.java
package com.persona.apipersona.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "comentarios")
public class Comentario {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String nombre;
    private String correo;
    private String asunto;
    private String comentario;
    private String fecha;

    public Comentario() {
    }

    public Comentario(int id, String nombre, String correo, String asunto, String comentario, String fecha) {
        this.id = id;
        this.nombre = nombre;
        this.correo = correo;
        this.asunto = asunto;
        this.comentario = comentario;
        this.fecha = fecha;
    }
    
    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public String getNombre() {
        return nombre;
    }
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    public String getCorreo() {
        return correo;
    }
    public void setCorreo(String correo) {
        this.correo = correo;
    }
    public String getAsunto() {
        return asunto;
    }
    public void setAsunto(String asunto) {
        this.asunto = asunto;
    }
    public String getComentario() {
        return comentario;
    }
    public void setComentario(String comentario) {
        this.comentario = comentario;
    }
    public String getFecha() {
        return fecha;
    }
    public void setFecha(String fecha) {
        this.fecha = fecha;
    }

    @Override
    public String toString() {
        return "Comentario [asunto=" + asunto + ", comentario=" + comentario + ", correo=" + correo + ", fecha=" + fecha
                + ", id=" + id + ", nombre=" + nombre + "]";
    }

    

    

    
}

File: front/front-asp/Models/_Comentario.cs
namespace front_asp.Models
{
    public class _Comentario
    {

        private int id;
        private String nombre;
        private String correo;
        private String asunto;
        private String comen;
        private String fecha;

        public _Comentario(int id, string nombre, string correo, string asunto, string comen, string fecha)
        {
            this.id = id;
            this.nombre = nombre;
            this.correo = correo;
            this.asunto = asunto;
            this.comen = comen;
            this.fecha = fecha;
        }

        public _Comentario() { 
        }

        public int Id { get => id; set => id = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Correo { get => correo; set => correo = value; }
        public string Asunto { get => asunto; set => asunto = value; }
        public string Comentario { get => comen; set => comen = value; }
        public string Fecha { get => fecha; set => fecha = value; }


        public override string ToString()
        {
            return $"{nameof(nombre)}: {nombre}, {nameof(correo)}: {correo} , {nameof(asunto)}: {asunto}, {nameof(comen)}: {comen}";
        }
    }


}
